site/en/docs/extensions/whatsnew/index.md

```diff
@@ -16,6 +16,14 @@ updated: 2022-02-17
 Check this page often to learn about changes to the Chrome extensions platform, its documentation,
 and related policy or other changes.
 
+### Chrome 102: {: #m102-registercontentscripts-main-world }
+
+April 14, 2022
+
+Dynamically registered content scripts can now specify a target world. See
```

Comment:
Is 'target world' something done in your product? If not, I would recommend replacing this a more easily understandable expression.

Reply:
Yep, it's an extension-specific concept. In short, extension content scripts normally run in a separate JS context from page script. We call this the extension's isolated world. The concept is explained in more detail in our [content scripts docs](https://developer.chrome.com/docs/extensions/mv3/content_scripts/#isolated_world).

I'm going to tweak the wording here to try to make it a little more clear and to link to that documentation.

Reply:
Thanks for the link.